src/site/content/en/handbook/content-checklist/index.md

```diff
@@ -10,9 +10,9 @@ description: |
 This is a checklist for web.dev content reviews. It includes critical high-level issues and lower-level issues that commonly appear in drafts.
 
 ## Basics
-1. Is the piece providing value for one or more of the web.dev [personas](/handbook/audience)?
-1. Do the title, subtitle, and description convey the value proposition?
-1. Is the piece 1,000 words or less—or is there a good reason for it to be longer?
+1. Does the piece provide value for one or more of the web.dev [personas](/handbook/audience)?
+1. Do the title, subtitle, and description convey a [value proposition](/handbook/audience/#value-propositions)?
+1. Is the piece 1,000 words or less? If it's longer, is there a good reason for it to be longer?
 1. Are all statements and advice accurate and aligned with best practices?
```

Comment:
link to best practices? How do contributors know that they're aligned to best practices?

Reply:
By using their domain knowledge, I was thinking. But I take the point that I think you're making: if that's the assumption, and we're not having the reviewer check against a reference, is this even a valuable question to ask? Maybe we cut it.

Reply:
This is entirely within domain knowledge meaning that the best judges of best practices are often the content contributors themselves, especially in articles about new features. Take service workers for instance. Many early articles were written by Jake Archibald. At that point he was the only source of best practices because he invented the thing.

One of the reasons we created WebFundamentals was that current best practices aren't written down yet. That's because best practices are often moving targets with the web. (I hope you'll take that at face value. This response is getting too long.)

This really needs to be (in my opinion) is a reminder to content contributors to put best practices in writing and to exemplify best practices in their code examples.

Reply:
I agree that this one is iffy but not sure how to make better, so gonna put off for now